app/views/sessions/show.html.erb

```diff
@@ -47,6 +49,10 @@
         </p>
       </div>
     </div>
+
+    <% if Flipper.enabled? :make_session_in_progress_button %>
```

Comment:
```suggestion
    <% if !@session.in_progress? && Flipper.enabled? :make_session_in_progress_button %>
```

Reply:
I thought about this, but then I thought it would be more consistent to just show it alway, since it's idempotent anyway.